### PR TITLE
Add node and yarn to base builder image

### PR DESCRIPTION
**Pull request by juliusv**
This is required for building the new React UI in
https://github.com/prometheus/prometheus/pull/5694

Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

**Comment by juliusv** [comment]
@sdurrheimer @simonpasquier

File: 1.13/base/Dockerfile [hunk 1]
```diff
@@ -13,6 +16,10 @@ RUN \
             gnupg \
             libsnmp-dev \
             make \
+        && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+        && echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list \
```
Comment (SuperQ) [inline 1.13/base/Dockerfile]: Put these at the top above the first `apt-get update`, then you can drop the next two commands and add it to the base list.

Comment (juliusv) [reply 1.13/base/Dockerfile]: So these lines require curl and gnupg, which is why we need to install the base packages above first...

File: 1.13/base/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,9 @@
 FROM        debian:stable
 MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
 
+ENV NODE_VERSION 10.15.2~dfsg-2
+ENV YARN_VERSION 1.19.0-1
```
Comment (SuperQ) [inline 1.13/base/Dockerfile]: Why do we need to specify a version? The upstream [install instructions](https://yarnpkg.com/en/docs/install#debian-stable) make it seem like it's fine to just pull in the latest stable automatically.

Comment (juliusv) [reply 1.13/base/Dockerfile]: Yeah, we can install it without explicit version constraint, but I thought it'd be wise to pin versions for anything that produces our final output binaries.

Comment (SuperQ) [reply 1.13/base/Dockerfile]: The only down side is that it creates an additional maintenance burden to keep the builder image up to date. We already spend some effort having to keep our Golang pinning up to date.

Comment (juliusv) [reply 1.13/base/Dockerfile]: Yep, that's the tradeoff. I don't care strongly either way... if you prefer removing the version constraints that's fine with me as well.

Comment (SuperQ) [reply 1.13/base/Dockerfile]: I'm in favor of "let's try unconstrained, and add a lock if we have issues later".

@simonpasquier might also have an opinion here.

Comment (simonpasquier) [reply 1.13/base/Dockerfile]: I'm fine with this option.

Comment (juliusv) [reply 1.13/base/Dockerfile]: Alrighty, removed the node & yarn version pinning.

**Comment by juliusv** [comment]
Good to merge now?

**Comment by simonpasquier** [comment]
LGTM

**Pull request merged**